.github/workflows/test-and-deploy.yml

```diff
@@ -82,6 +82,7 @@ jobs:
       - name: Publish package to NuGet
         run: |
           make release
+          dotnet nuget sign foo.nupkg --certificate-fingerprint 34bb5309a34488f8bac21af3169716293da2299e5ed24a07992fc557867ab809
```

Comment:
Please put above signing code in secrets.
Provide path to twilio-csharp nupkg file. Path is ===>    bin/Release/<file-name>
foo.nupkg -> correct the name.

Reply:
Added signing key in secrets, provided path to certificate file and corrected the nupkg file name